dvc/external_repo.py

```diff
@@ -45,117 +36,49 @@ def __init__(self, url, rev, cause):
         )
 
 
-class ExternalRepo(object):
-    REPOS_DIR = "repos"
-
-    PARAM_URL = "url"
-    PARAM_VERSION = "rev"
-
-    SCHEMA = {Optional(PARAM_URL): str, Optional(PARAM_VERSION): str}
+def _install(url=None, rev=None, rev_lock=None, cache_dir=None):
+    import git
+    from dvc.repo import Repo
 
-    def __init__(self, dvc_dir, **kwargs):
-        self.repos_dir = os.path.join(dvc_dir, self.REPOS_DIR)
-        self.url = kwargs[self.PARAM_URL]
+    _path = tempfile.mkdtemp("dvc-repo")
```

Comment:
where will it be created? the same fs or /tmp?

Reply:
tmp. We only care about fs for the cache dir, which is configures separately.

Reply:
make sense 👍